compiler/bindings/python/IREECompilerDialectsModule.cpp

```diff
@@ -501,4 +501,12 @@ NB_MODULE(_ireeCompilerDialects, m) {
       "Infers the structure of an attention operation from affine indexing "
       "maps.",
       py::arg("q"), py::arg("k"), py::arg("v"), py::arg("o"));
+
+  iree_codegen_module.def(
+      "isa_attention_op",
+      [](MlirOperation op) -> bool {
+        return ireeCodegenMlirOperationIsACodegenAttentionOp(op);
+      },
```

Comment:
Do we need the lambda here? If the types match, you shouldn't need a wrapper around `ireeCodegenMlirOperationIsACodegenAttentionOp`